dart/test/utils_test.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test/test.dart';
+import 'package:vm_service_lib/utils.dart';
+
+@TestOn('vm')
```

Comment:
this test should run everywhere, not just on the VM, right?

Reply:
Good point, I've removed that.